custom_components/alexa_media/__init__.py

```
@@ -503,33 +505,42 @@ async def async_update_data() -> Optional[AlexaEntityData]:
+                    else:
+                        _LOGGER.debug(
+                            "Network discovery succeeded. Processing results."
+                        )
+                        # Only process this once after success
+                        hass.data[DATA_ALEXAMEDIA]["accounts"][email][
+                            "should_get_network"
+                        ] = False
+                        # Disregard entities_to_monitor results since we now have full network details
+                        if entities_to_monitor:
+                            optional_task_results.pop()
+                            entities_to_monitor.clear
```

Ruff (0.14.5)

Found useless expression. Either assign it to a variable or remove it.

(B018)